app/api/attendees.py

```diff
@@ -149,6 +151,19 @@ def before_update_object(self, obj, data, kwargs):
                 if obj.checkin_times and data['checkin_times'] not in obj.checkin_times.split(","):
                     data['checkin_times'] = '{},{}'.format(obj.checkin_times, data['checkin_times'])
 
+        if 'is_checked_out' in data and data['is_checked_out']:
+            if 'checkout_times' not in data:
+                raise UnprocessableEntity(
+                    {'pointer': '/data/attributes/checkout_times'},
+                    "Check out time missing while trying to check out"
+                    "attendee")
+            else:
+                if obj.checkout_times and data['checkout_times'] not in \
+                        obj.checkout_times.split(","):
+                    data['checkout_times'] = '{},{},{}'.format(
```

Comment:
I didn't get the point of last one. If you are expecting a checkout_time from the frontend then why are you appending both the user one and the current timestamp ?

Reply:
Since we just established that it will be a read-only value, we can eliminate the checkout time sent by the FE?

Reply:
Yeah we can.